rulesets/resources/ES_ZERHruleset.rb

```diff
@@ -496,7 +496,7 @@ def self.set_enclosure_floors_reference(orig_bldg, new_bldg)
       floor_ufactor = lookup_reference_value('floors_over_uncond_spc_ufactor', subtype)
       floor_ufactor = lookup_reference_value('floors_over_uncond_spc_ufactor') if floor_ufactor.nil?
 
-      if orig_floor.is_thermal_boundary && (not (multifamily_adjacent_locations - [HPXML::LocationOtherNonFreezingSpace]).include?(orig_floor.exterior_adjacent_to))
+      if orig_floor.is_thermal_boundary && (not [HPXML::LocationOtherHousingUnit, HPXML::LocationOtherHeatedSpace].include?(orig_floor.exterior_adjacent_to))
```

Comment:
Since the logic is a bit indirect... can you add a comment explaining what types of floors are meant to be covered by this?